src/Util/Search.php

```diff
@@ -486,6 +491,56 @@ private function getProjectDirectorInfo(array $aggregations): array
         return $projectDirectorInfo;
     }
 
+    /**
+     * Get the funder aggregations for the query.
+     *
+     * @param Query $query The query built based on the search terms and parameters.
+     *
+     * @return array
+     */
+    public function getFunderAggregations(Query $query): array
+    {
+        $userPaginator = $this->getPaginator($query);
+        $funderBucket = array_column(
+            $this->findKey($userPaginator->getAdapter()->getAggregations(), 'funderId')['buckets'],
+            'doc_count',
+            'key'
+        );
+        return $this->getFunderInfo($funderBucket);
+    }
+
+    /**
+     * Get funders information for the aggregations.
+     *
+     * @param array $aggregations Aggregations for each funder id.
+     *
+     * @return array
+     */
+    private function getFunderInfo(array $aggregations): array
+    {
+        $fundersInfo = array();
+        $funders = $this->entityManager
+            ->getRepository(Funder::class)
+            ->findBy(array('id' => array_keys($aggregations)));
+        foreach ($funders as $funder) {
+            $fundersInfo[$funder->getId()] = array(
+                'id' => $funder->getId(),
+                'name' => $funder->getName(),
+                'count' => $aggregations[$funder->getId()]
+            );
+        }
+        //Sorting based on highest count
+        $array_column1 = array_column($fundersInfo, 'count');
+        $array_column2 = array_column($fundersInfo, 'name');
```

Comment:
Maybe these could use better names, but it's not too important.

Reply:
Yes, but i felt this was anyways going to be replaced with multi-search